stabilizer.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+"""
+Author: Vertigo Designs, Ryan Summers
+
+Description: Provides an API for controlling Stabilizer over Miniconf (MQTT).
+"""
+import argparse
+import asyncio
+import json
+
+from gmqtt  import Client as MqttClient
+
+
+def parse_value(value):
```

Comment:
I would like to ideally move `parse_value` and `MiniconfApi` over to the `miniconf` repository in the future and release it as a package via pip, but it's existing here as an initial prototype. Let me know what you think @jordens

Reply:
Just `json.loads()`?

Reply:
But Yes. Move `Miniconf` to `miniconf` when convenient and ~stable.

Reply:
Ah, json-loads is a much simpler approach. I don't know why that slipped my mind. Good catch